component/src/main/java/org/wso2/extension/siddhi/execution/geo/internal/impl/GeoCoordinateResolverHolder.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.extension.siddhi.execution.geo.internal.impl;
+
+import org.wso2.extension.siddhi.execution.geo.api.GeoCoordinateResolver;
+
+/**
+ * A singleton class to initialize extension.
+ */
+public class GeoCoordinateResolverHolder {
+
+    private static GeoCoordinateResolverHolder geoCoordinateResolverHolder;
```

Comment:
This is a static class, 
    private static GeoCoordinateResolverHolder geoCoordinateResolverHolder = new GeoCoordinateHolder();
after which use setGeoCoordinateResolver

Reply:
Fixed with commit : 5b45006a43d923b4aa80b9aa828e959af267de86